requirements.txt

```diff
@@ -22,7 +22,7 @@ pycryptodome==3.9.8
 pyOpenSSL==19.1.0
 pytest-html==2.0.1
 pytest==6.2.2
-pyyaml==5.3
+pyyaml==5.4
```

Comment:
Is it necessary to change `pyyaml` version?

Reply:
It is because of a problem when trying to install the dependencies with that version, here is a PR with the same change pointing to 4.2:
https://github.com/wazuh/wazuh-qa/pull/1736